### DISCUSSION TITLE
How do I make a Kata in Python?

### DISCUSSION THREAD

**Discussion by BinaryAir** [codewars/codewars.com#2631] [Q&A]
I don't understand, it shows me a bug in my solution and I don't understand what bugs there are.
+
My solution works smoothly on Jupyter
+
What do I give in the test cases and how do I type my test cases?
+
What do preloaded options do

**Comment by hobovsky** [comment]
I am not sure if I understand you correctly, but as far as I get it, you have a problem solving a kata, right? If so, there are some places which could be helpful for you:

 - You can read [Solution troubleshooting guide](https://docs.codewars.com/training/troubleshooting)
 - You can ask questions on kata discourse.
 - You can ask for help on Codewars Discord server.
